ledger/participant-state-errors/src/main/scala/com/daml/ledger/participant/state/kv/errors/KVErrors.scala

```diff
@@ -1,11 +1,9 @@
 // Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Nit:
The naming of this class is incorrect, as it should rather be `KvErrors` to follow [correct camel-casing of abbreviations](https://docs.scala-lang.org/style/naming-conventions.html).